Remotion/Web/Development.WebTesting/WebTestSetUpFixtureHelper.cs

```diff
@@ -107,6 +117,51 @@ private void HostWebApplication ()
       _hostingStrategy.DeployAndStartWebApplication();
     }
 
+    private void VerifyWebApplicationStarted (string webApplicationRoot, TimeSpan applicationPingTimeout)
+    {
+      var stopwatch = Stopwatch.StartNew();
+      
+      var webRequest = (HttpWebRequest)HttpWebRequest.Create (webApplicationRoot);
+      webRequest.Method = WebRequestMethods.Http.Head;
+      webRequest.AllowAutoRedirect = true;
+      
+      //Just setting this to some value for the first run
+      var statusCode = HttpStatusCode.NotFound;

```

Comment:
If you just need _some_ value I **think** you can do `HttpStatusCode statusCode = default;` When I tried it out, the variable was initialized to 0. which does not correspond to any status code so I have no idea if it will behave as expected, but it would spare you the comment.

Reply:
Good idea!